charts/consul/test/unit/client-daemonset.bats

```diff
@@ -2011,6 +2011,7 @@ rollingUpdate:
   local actual=$(helm template \
       -s templates/client-daemonset.yaml  \
       --set 'client.enabled=true' \
+      --set 'global.enableConsulNamespaces=true' \
```

Comment:
You will see this change throughout. `global.enableConsulNamespaces` must be `true` if `global.adminPartitions.enabled=true`.